draft-ietf-tls-tls13.md

```diff
@@ -2749,38 +2749,45 @@ servers MUST process the client's ClientHello and then immediately
 send the ServerHello, rather than waiting for the client's
 EndOfEarlyData message.
 
+#### Age and Lifetime {#age-lifetime}
+
+When a client create a NewSessionTicket message, the ticket_age is
+calculated as follows:
+
+       ticket_age = client_create_psk_time - client_recv_nst_time
```

Comment:
"client_create_psk_time" needs to be defined.  I assume the intent was the time when the PreSharedKeyExtension is generated, but that is not clear from just the name.